src/App.tsx

```diff
@@ -1,14 +1,91 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+function filterBySelect(
+  selectValue: string,
+  todos: Todo[],
+) {
+  switch (selectValue) {
+    case 'active':
```

Comment:
move filter cases to enum

Reply:
Not resolved. If you have any questions - feel free to ask them in `fe_chat`